### DISCUSSION TITLE
I need change output bitrate

### DISCUSSION THREAD

**Discussion by PenguinEino** [audiamus/AaxAudioConverter#73] [Q&A] [answered]

**Comment by PenguinEino** [comment]
Can i change output bitrate?

**Comment by audiamus** [comment, marked as answer]
You can lower the bit rate via the Basic Settings, Conversion tab: 
![image](https://user-images.githubusercontent.com/47717419/117110421-619f8800-ad86-11eb-8859-e4444b29bb37.png)

See the manual for details.
